app/src/main/java/org/mifos/mobilebanking/api/local/PreferencesHelper.java

```diff
@@ -96,12 +110,13 @@ public void setUserId(long id) {
     }
 
     public String getTenant() {
-        return getString(TENANT, "default");
+        return getString(context.getString(R.string.pref_base_tenant_key),
+                SelfServiceInterceptor.DEFAULT_TENANT);
     }
 
     public void setTenant(String tenant) {
         if (!TextUtils.isEmpty(tenant)) {
-            putString(TENANT, tenant);
+            putString(context.getString(R.string.pref_base_tenant_key), tenant);
```

Comment:
Don't use key value name from strings.xml, please define key value on top of class like these https://github.com/openMF/mifos-mobile/pull/716/files#diff-4afc21547670976db853693ae9f5d17cR29